scripts/risk_pipeline/data_prep/get/reservoir_nft_data.py

```diff
@@ -0,0 +1,149 @@
+import requests
+import pandas as pd
+import argparse
+import time
+import os
+import sys
+sys.path.insert(0, os.getcwd()+'/scripts/risk_pipeline')
+from helpers import helpers  # noqa: E402
+
+FLOORS = "https://api.reservoir.tools/events/collections/floor-ask/v1"
+SALES = "https://api.reservoir.tools/sales/v4"
+
+
+def epoch_time(ts_string):
+    time_struct = time.strptime(ts_string, "%Y-%m-%d-%H-%M")
+
+    # Convert the time struct to Unix time and return
+    return int(time.mktime(time_struct))
+
+
+def get_params():
+    """
+    Get parameters from command line
+    """
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        '--collection_addr', type=str,
+        help=('Contract address of the collection.'
+              'Ex, for BAYC: 0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D')
+    )
+    parser.add_argument(
+        '--collection_name', type=str,
+        help='Name of the collection. Ex: BAYC'
+    )
+    parser.add_argument(
+        '--start_time', type=str,
+        help='Start time for data pull. Ex: 2016-06-01-00-00'
+    )
+    parser.add_argument(
+        '--end_time', type=str,
+        help='End time for data pull. Ex: 2016-06-01-00-00'
+    )
+    parser.add_argument(
+        '--type', type=str,
+        help='sales or floors. Ex: sales'
+    )
+
+    args = parser.parse_args()
+    start_time = epoch_time(args.start_time)
+    end_time = epoch_time(args.end_time)
+
+    return (args.collection_addr, args.collection_name,
+            start_time, end_time, args.type)
+
+
+def get_file_name(cname, start, end, type):
+    return f"{cname}_{type}_{start}_{end}.csv"
+
+
+def query_data(caddr, start, end, type):
+
+    # Set query parameters
+    params = {
+        'collection': caddr.lower(),
+        'startTimestamp': start,
+        'endTimestamp': end,
+        'limit': 1000
+    }
+
+    # Pull data
+    if type == 'sales':
+        url = SALES
+    elif type == 'floors':
+        url = FLOORS
+    response = requests.get(url, params)
+    response_dicts = [response.json()]
+    i = 0
+    while response_dicts[i]['continuation']:
+        print(i)
+        params['continuation'] = response_dicts[0]['continuation']
+        response = requests.get(url, params)
+        response_dicts.append(response.json())
+        if response_dicts[-1] == response_dicts[-2]:
+            response_dicts = response_dicts[:-1]
+            break
+        i += 1
+
+    # Arrange data in pandas df
+    if type == 'sales':
+        dfs = [pd.json_normalize(d['sales']) for d in response_dicts]
+        df = pd.concat(dfs, ignore_index=True)
+        # TODO: Catch when no data is returned
+        price_df = df[['price.currency.name', 'price.amount.decimal',
```

Comment:
would be better to collect all expected columns in one place as symbolic constants, rather than using a bunch of magic strings that nobody knows anything about